pkg/apis/stork/v1alpha1/clusterdomains.go

```diff
@@ -0,0 +1,75 @@
+package v1alpha1
+
+import (
+	meta "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const (
+	// ClusterDomainsStatusResourceName is name for "clusterdomainsstatus" resource
+	ClusterDomainsStatusResourceName = "clusterdomainsstatus"
```

Comment:
Define the plural for this too. You'll need it when registering the CRD

Reply:
done